Group_Constitution

```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
```

Comment:
Thanks for getting this going again @linamnt! 

The name we are likely going with is "U of T Scientific Coders" or "University of Toronto Scientific Coders" (see https://github.com/UofTCoders/studyGroup/pull/40 for the discussion on it).